metaflow/mflog/__init__.py

```diff
@@ -37,6 +37,13 @@
             ' >> $MFLOG_STDOUT; echo $1; '\
      ' }' % TASK_LOG_SOURCE
 
+BASH_MFLOG_KFP =\
+    'mflog(){{ '\
+        'T=$(date -u -Ins|tr , .); '\
+        'echo \"[MFLOG|0|${{T:0:26}}Z|%s|$T]$1\"'\
+            ' >> $MFLOG_STDOUT; echo $1; '\
+     ' }}' % TASK_LOG_SOURCE
```

Comment:
Double bracket and slightly different symbol escapes are needed to avoid bash command being unexpectedly formatted by python.

Reply:
Example log line:
```
[MFLOG|0|2021-07-29T08:04:10.669355Z|task|8309088c-d273-4998-a7c5-a181009c91fc]Task is starting.
```
See the sample datastore for more examples 🙂.

Reply:
Please add as comments, thanks!